### PR TITLE
Added script for database and ASCII Art

### PR DESCRIPTION
**Pull request by vintej**

### CODE REVIEW

File: cassandra_slo/mininet/topology_slo.py [hunk 1]
```diff
@@ -73,7 +78,7 @@ def createTopology(self):
                              privateDirs=cassandra_private_dirs)
         host6 = self.addHost('h6', ip="100.0.0.16/24",
                              privateDirs=cassandra_private_dirs)
-
+
```
Comment (juliuscc) [inline cassandra_slo/mininet/topology_slo.py]: wow

File: turtles_art [hunk 2]
```diff
@@ -0,0 +1,29 @@
+====================================================================================
```
Comment (juliuscc) [inline turtles_art]: Very impressive! This boosts my moral

File: cassandra_slo/mininet/topology_slo.py [hunk 3]
```diff
@@ -50,9 +50,14 @@ def __init__(self):
         self.startCassandra('h1')
         self.startCassandra('h2')
 
-        # Make sure that everything is set up
+    # Setup Database
```
Comment (juliuscc) [inline cassandra_slo/mininet/topology_slo.py]: wow

File: turtles_art (hunk 2 above)
Comment (ShubhankerS) [reply turtles_art]: +1

**Pull request merged**